styles/global-blocks.css

```diff
@@ -844,4 +844,8 @@ a.card:hover {
         margin-block-end: var(--spacing-global-block-end-large-screen);
       }
     }
+
+  > .grid-container .grid-item:has(.section-header) {
+    align-self: center;
+  }
```

Comment:
can we write a poem with selectors

Reply:
this comment will not get the mass attention it deserves

Reply:
I'll leave it up for you to admire my brilliance during your second pass.

Reply:
thank you